wp-content/themes/currentorg/less/current-ltw-projects.less

```diff
@@ -117,27 +162,35 @@
 			margin: 0;
 			vertical-align: initial;
 		}
+		summary::-webkit-details-marker {
+			display: none;
+		}
 	}
 }
 .projects-list {
-	border: 1px solid @grey-d;
-	padding: @common-whitespace * 0.5;
 }
 .projects-list-item {
-	border: 1px solid @grey-d;
+	border: 1px solid @grey-d!important;
 	padding: @common-whitespace * 0.5;
+	background: @white;
 	& + .projects-list-item {
-		margin-top: @common-whitespace * 0.5;
+		margin-top: 0.5rem;
+	}
+	&.active {
+		border-left: 6px solid @azure!important;
 	}
 }
 .projects-single-layout {
 	border: 1px solid @grey-d;
+	border-left: 6px solid @azure;
 	padding: @common-whitespace * 0.5;
-	width: 95%;
+	width: 100%;
```

Comment:
```diff
+	box-sizing: border-box;
````

Reply:
Added in https://github.com/INN/umbrella-currentorg/pull/144/commits/085858c39df3653dac3e16076f410c57e53889e4